packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -33,24 +33,24 @@ export function comboboxTemplate<T extends FASTCombobox>(
                         aria-disabled="${x => x.ariaDisabled}"
                         aria-expanded="${x => x.ariaExpanded}"
                         aria-haspopup="listbox"
-                        class="selected-value"
-                        part="selected-value"
+                        class="field"
+                        part="field"
                         placeholder="${x => x.placeholder}"
                         role="combobox"
                         type="text"
                         ?disabled="${x => x.disabled}"
                         :value="${x => x.value}"
                         @input="${(x, c) => x.inputHandler(c.event as InputEvent)}"
                         @keyup="${(x, c) => x.keyupHandler(c.event as KeyboardEvent)}"
-                        ${ref("control")}
+                        ${ref("field")}
                     />
-                    <div class="indicator" part="indicator" aria-hidden="true">
-                        <slot name="indicator">
-                            ${options.indicator ?? ""}
-                        </slot>
-                    </div>
                 </slot>
                 ${endSlotTemplate(options)}
+                <div class="open-close-icon" part="open-close-icon" aria-hidden="true">
+                    <slot name="open-close-icon">
+                        ${options.openCloseIcon ?? ""}
+                    </slot>
+                </div>
```

Comment:
I'm not sure that we are getting an improvement semantically from `indicator` here to `open-close-icon` besides verbosity - is there a reason this change is seen as an improvement?

Reply:
Additionally, I'm not sure this should be moved in the DOM.

Reply:
The convention for icon naming is "indicator" is used for something that indicates a persistent state, like "checked". "icon" is used for something that represents an action like step up / down or a transient state like a listbox being open. This seems to be the strongest for consistency between the two uses. The "icon" may not always _indicate_ something because it may not change, like for open and closed in this component.

Reply:
Regarding moving within the dom, like other components like Accordion item or menu, the "start" and "end" slots are primarily for content, which is all related, like an icon before or after the text content. Having the chevron (typically) _before_ the "end" slot breaks that relationship, however if someone needs to adjust this they can change the css grid column.

Reply:
> The convention for icon naming is "indicator" is used for something that indicates a persistent state, like "checked". "icon" is used for something that represents an action like step up / down or a transient state like a listbox being open. This seems to be the strongest for consistency between the two uses. The "icon" may not always _indicate_ something because it may not change, like for open and closed in this component.

I think this _not_ being persistent is again a matter of semantics that's not quite universal. The current native select doesn't show the indicator in an "expanded" state, so there is just one. There's only a single slot in the Open UI Select Menu approach and I think we should map to that per our goals.

In terms of moving the DOM, based on the select menu work happening, the resolved approach is to have a slot specific for the indicator to allow it to be replaced. I think that's what we have now with the above (replace indicator w/o having to replace the entire field to do so, so I think after consulting Open UI we are good on that.